### PR TITLE
fix(list items): rename info & infoState properties

### PR DESCRIPTION
**Pull request by fifoosid**
Part of #3107

BREAKING_CHANGE: ```info``` and ```infoState``` properties of ```StandardListItem``` have been deprecated in favour of ```additionalText``` and ```AdditionalTextState``` properties

BREAKING_CHANGE: ```info``` and ```infoState``` properties of ```SuggestionItem``` have been deprecated in favour of ```additionalText``` and ```AdditionalTextState``` properties

BREAKING_CHANGE: ```info``` and ```infoState``` properties of ```TreeItem``` have been deprecated in favour of ```additionalText``` and ```AdditionalTextState``` properties

BREAKING_CHANGE: ```info``` and ```infoState``` properties of ```TreeListItem``` have been deprecated in favour of ```additionalText``` and ```AdditionalTextState``` properties

### CODE REVIEW

File: packages/main/src/InputPopover.hbs [hunk 1]
```diff
@@ -98,9 +98,9 @@
 				<ui5-li-suggestion-item
 					image="{{this.image}}"
 					icon="{{this.icon}}"
-					info="{{this.info}}"
+					additionalText="{{this.additionalText}}"
```
Comment (nnaydenow) [inline packages/main/src/InputPopover.hbs:101]: additional-text

File: packages/main/src/StandardListItem.hbs [hunk 2]
```diff
@@ -7,8 +7,8 @@
 			<div class="ui5-li-description-info-wrapper">
 				<span part="description" class="ui5-li-desc">{{description}}</span>
 
-				{{#if info}}
-					<span part="info" class="ui5-li-info">{{info}}</span>
+				{{#if additionalText}}
+					<span part="additionalText" class="ui5-li-info">{{additionalText}}</span>
```
Comment (nnaydenow) [inline packages/main/src/StandardListItem.hbs:11]: I'm not familiar with part but should we keep it value in camelCase style?

**Review by nnaydenow — COMMENTED** [review]
At all it looks okey for me. We can add someone from @SAP/ui5-webcomponents-topic-rl to approve the changes in input files

File: packages/main/src/StandardListItem.hbs (hunk 2 above)
Comment (ilhan007) [reply packages/main/src/StandardListItem.hbs:11]: @nnaydenow we decided to write the CSS Shadow Parts in kebab-case

Comment (nnaydenow) [reply packages/main/src/StandardListItem.hbs:11]: @ilhan007 we can submit this when @fifoosid fix this

**Review by ilhan007 — APPROVED** [review]

File: packages/main/src/themes/ListItem.css [hunk 3]
```diff
@@ -32,27 +32,27 @@
...
 :host([active][actionable]) .ui5-li-title,
 :host([active][actionable]) .ui5-li-desc,
 :host([active][actionable]) .ui5-li-info {
 	color: var(--sapList_Active_TextColor);
 }
 
-/* [ui5-li]: infoState */
-:host([info-state="Warning"]) .ui5-li-info {
+/* [ui5-li]: additionalTextState */
+:host([additional-text-state="Warning"]) .ui5-li-info {
```
Comment (ilhan007) [inline packages/main/src/themes/ListItem.css:43]: .ui5-li-info class should be also renamed perhaps

File: packages/main/src/themes/TreeListItem.css [hunk 4]
```diff
@@ -89,24 +89,24 @@
     color: var(--sapList_Active_TextColor);
 }
 
-/* [ui5-li]: infoState */
-:host([info-state="Warning"]) .ui5-li-tree-info {
+/* [ui5-li]: additionalTextState */
+:host([additional-text-state="Warning"]) .ui5-li-tree-info {
```
Comment (ilhan007) [inline packages/main/src/themes/TreeListItem.css:93]: .ui5-li-tree-info class should be also renamed perhaps, but check if it is going to be removed by this PR https://github.com/SAP/ui5-webcomponents/pull/3280

**Review by ilhan007 — CHANGES_REQUESTED** [review]
Check some css class naming in relation to the prop name change, described inline

**Comment by nnaydenow** [comment]
For me it looks okey now.

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**